src/data/location-data.js

```diff
@@ -0,0 +1,152 @@
+export default {
+  locationData: [
+    {
+      status: 'success',
+      data: {
+        city: 'Thornton',
+        state: 'Colorado',
+        country: 'USA',
+        location: {
+          type: 'Point',
+          coordinates: [
+            -104.97192,
+            39.86804
+          ]
+        },
```

Comment:
so glad you sorted out the weird error the linter was giving us for this data file, but still curious if we'll need to readdress the formatting when we parse data from the api